### PR TITLE
Fix models fields

### PR DESCRIPTION
**Pull request by gdalmau**

### CODE REVIEW

File: sii/models.py [hunk 1]
```diff
@@ -23,7 +29,14 @@ class Cabecera(Schema):
 
 class PeriodoImpositivo(Schema):
     Ejercicio = fields.String(required=True)  # TODO validate Año en formato 'YYYY'
-    Periodo = fields.String(required=True)  # TODO validate Enumeration '01' para enero, '02' para febrero, ..., '0A' Periodicidad anual
+    Periodo = fields.String(required=True)
```
Comment (ecarreras) [inline sii/models.py]: You can use [OneOf](https://marshmallow.readthedocs.io/en/latest/api_reference.html#marshmallow.validate.OneOf) to do this validation as following:
```python
Periodo = fields.String(validate=validate.OneOf(PERIODO_VALUES))
```

File: sii/models.py [hunk 2]
```diff
@@ -62,7 +75,7 @@ class NoExenta(Schema):
 
     @validates('TipoNoExenta')
     def validate_tipo_no_exenta(self, value):
-        if value not in ['S1', 'S2']:
+        if value not in TIPO_NO_EXENTA_VALUES:
```
Comment (ecarreras) [inline sii/models.py]: Use `validateOneOf` as mentioned in previous comment

File: sii/models.py (hunk 1 above)
Comment (gdalmau) [inline sii/models.py]: Done in https://github.com/gisce/sii/pull/10

File: sii/models.py (hunk 2 above)
Comment (gdalmau) [inline sii/models.py]: Done in https://github.com/gisce/sii/pull/10

**Pull request merged**